app/models/request/sample_compound_aliquot_transfer.rb

```diff
@@ -108,13 +106,28 @@ def _create_compound_sample(study, component_samples)
   end
 
   # Default study that the new compound sample will use
+  # Uses the one from the request if it's present,
+  # otherwise, the one from the source aliquots if it's consistent.
   def _default_compound_study
-    _studies.first
+    _initial_study ||
+      begin
+        raise MULTIPLE_STUDIES_ERROR_MSG if _studies.count > 1
```

Comment:
It might make sense to raise a custom exception class which we can then rescue and return a useful error message to the end user. Could even point them to request additional sequencing as a potential solution then.

Reply:
Actually, isn't this validation too strict? It's looking at the studies on *All* aliquots in the receptacle, not just those that will compounded together.